libp2p/Cargo.toml

```diff
@@ -94,6 +94,7 @@ futures-timer = "3.0.2" # Explicit dependency to be used in `wasm-bindgen` featu
 getrandom = "0.2.3" # Explicit dependency to be used in `wasm-bindgen` feature
 instant = "0.1.11" # Explicit dependency to be used in `wasm-bindgen` feature
 
+libp2p-allow-block-list = { version = "0.1.0", path = "../misc/allow-block-list" }
```

Comment:
Needs a bump of `libp2p` to `v0.51.2`, otherwise this will not be released with the next patch release round.

Reply:
Sure!